script.js

```diff
@@ -29,11 +29,14 @@ const compliments = [
 	'You are making a difference.',
 	'You bring out the best in other people.',
 ];
-
+const message=document.querySelector("#subtitle");
```

Comment:
add white space before and after "=" sign

Reply:
can you name this variable subtitle instead?

Reply:
ok